server/src/index.js

```diff
@@ -26,8 +26,10 @@ const users = cache.create(async (user) => {
   return {...u.profile, id: user}
 })
 const channels = cache.create(async (channel) => {
-  const c = await web.conversations.info({channel})
-  return c.channel
+  const c = await web.conversations.info({channel}).catch((error) => {
```

Comment:
I've thought about doing the same for users, but

1. Users can't be deleted, just archived
2. It would be more complicated to properly handle missing users (profile pictures, @mentions, etc.)

So I think it's ok to fail fast & hard if there is some problem with users, and fix it when we know what it is.